cilium/tests/common.py

```diff
@@ -137,4 +137,74 @@
+    "cilium.operator.ipam.deficit_resolver.folds",
+    "cilium.operator.ipam.deficit_resolver.queued.count",
+    "cilium.operator.ipam.deficit_resolver.latency.seconds.sum",
+    "cilium.operator.ipam.deficit_resolver.latency.seconds.count",
+    "cilium.operator.ipam.deficit_resolver.latency.seconds.bucket",
+    "cilium.operator.ipam.ips",
+    "cilium.operator.ipam.k8s_sync.duration.seconds.sum",
+    "cilium.operator.ipam.k8s_sync.duration.seconds.count",
+    "cilium.operator.ipam.k8s_sync.duration.seconds.bucket",
+    "cilium.operator.ipam.k8s_sync.folds",
+    "cilium.operator.ipam.k8s_sync.queued.count",
+    "cilium.operator.ipam.k8s_sync.latency.seconds.sum",
+    "cilium.operator.ipam.k8s_sync.latency.seconds.count",
+    "cilium.operator.ipam.k8s_sync.latency.seconds.bucket",
+    "cilium.operator.ipam.nodes",
+    "cilium.operator.ipam.resync.duration.seconds.sum",
+    "cilium.operator.ipam.resync.duration.seconds.count",
+    "cilium.operator.ipam.resync.duration.seconds.bucket",
+    "cilium.operator.ipam.resync.count",
+    "cilium.operator.ipam.resync.queued.count",
+    "cilium.operator.ipam.resync.folds",
+    "cilium.operator.ipam.resync.latency.seconds.sum",
+    "cilium.operator.ipam.resync.latency.seconds.count",
+    "cilium.operator.ipam.resync.latency.seconds.bucket",
+    "cilium.operator.ipam.allocation_ops.count",
+    "cilium.operator.ipam.interface_creation_ops.count",
+    "cilium.operator.ipam.release_ops.count",
+]
+
+# Not available in test metric fixtures
+ADDL_OPERATOR_AWS_METRICS = [
+    "cilium.operator.ec2.api.rate_limit.duration.seconds.sum",
+    "cilium.operator.ec2.api.rate_limit.duration.seconds.count",
+    "cilium.operator.ec2.api.rate_limit.duration.seconds.bucket",
+    "cilium.operator.eni_ec2.rate_limit.duration.seconds.sum",
+    "cilium.operator.eni_ec2.rate_limit.duration.seconds.count",
+    "cilium.operator.eni_ec2.rate_limit.duration.seconds.bucket",
+    "cilium.operator.ipam.api.duration.seconds.sum",
+    "cilium.operator.ipam.api.duration.seconds.count",
+    "cilium.operator.ipam.api.duration.seconds.bucket",
+    "cilium.operator.ipam.api.rate_limit.duration.seconds.sum",
+    "cilium.operator.ipam.api.rate_limit.duration.seconds.count",
+    "cilium.operator.ipam.api.rate_limit.duration.seconds.bucket",
+]
+
+ADDL_OPERATOR_METRICS = [
+    "cilium.operator.identity_gc.entries",
+    "cilium.operator.identity_gc.runs",
+    "cilium.operator.num_ceps_per_ces.sum",
+    "cilium.operator.num_ceps_per_ces.count",
+    "cilium.operator.num_ceps_per_ces.bucket",
+    "cilium.operator.ces.queueing_delay.seconds.sum",
+    "cilium.operator.ces.queueing_delay.seconds.count",
+    "cilium.operator.ces.queueing_delay.seconds.bucket",
+    "cilium.operator.ces.sync_errors.count",
+]
+
+OPTIONAL_METRICS = [
+    "cilium.datapath.errors.count",
+    "cilium.endpoint.count",
+    "cilium.identity.count",
+    "cilium.policy.count",
 ]
```

Comment:
```suggestion
OPTIONAL_METRICS = {
    "cilium.datapath.errors.count",
    "cilium.endpoint.count",
    "cilium.identity.count",
    "cilium.policy.count",
}
```
Since you're using this just for checking if a metric is in the list. If you define it as a set lookout is faster. Not that it matters in this case but is a think to keep in mind

Reply:
good tip, thanks!